src/utils/featheryClient/index.ts

```diff
@@ -574,11 +574,27 @@ export default class FeatheryClient extends IntegrationClient {
       data.field_values = fieldValues;
     }
 
-    return this._fetch(`${API_URL}custom_request/`, {
+    const url = `${API_URL}custom_request/`;
```

Comment:
const run = () => this._fetch(url, options);
if (typeof payload === 'string' && payload.method === 'GET') {
  return run().then(response...)
}

return this.offlineRequestHandler...